arangod/RestServer/FlushFeature.cpp

```diff
@@ -129,8 +124,6 @@ arangodb::Result FlushFeature::releaseUnusedTicks(size_t& count, TRI_voc_tick_t&
     TRI_TerminateDebugging("crashing before syncing min tick");
   }
 
-  engine->waitForSyncTick(minTick);
```

Comment:
I'd prefer to leave a comment that WAL tick has to be synced prior releasing it, again for the future

Reply:
Ok, did!